mlflow/sklearn/utils.py

```diff
@@ -310,7 +310,12 @@ def plot_confusion_matrix(*args, **kwargs):
                 "axes.labelsize": 10.0,
             }
         ):
-            return sklearn.metrics.plot_confusion_matrix(*args, **kwargs)
+            if Version(sklearn.__version__) >= Version("1.0"):
```

Comment:
Can we put `Version(sklearn.__version__) >= Version("1.0")` in a variable (e.g. `is_plotting_function_deprecated`) and reuse it?